bicepconfig.json

```diff
@@ -6,11 +6,7 @@
     },
     "analyzers": {
         "core": {
-            "rules": {
-                "no-deployments-resources": {
-                    "level": "off" // Reason: The telemetry resource only has a single output and is self-contained in a deployment
-                }
-            }
+            "rules": {}
```

Comment:
Can we have just one pipeline badge?

Reply:
Sure. I wouldn't know why as these are Bicep Linting rules that don't matter to any of our logic, but let me find a very small module 😄

Reply:
[![avm.res.analysis-services.server](https://github.com/AlexanderSehr/bicep-registry-modules/actions/workflows/avm.res.analysis-services.server.yml/badge.svg?branch=users%2Falsehr%2FremoveDeploymentSuppressionFromBicepConfig&event=workflow_dispatch)](https://github.com/AlexanderSehr/bicep-registry-modules/actions/workflows/avm.res.analysis-services.server.yml)